pkg/config/default.go

```diff
@@ -233,8 +233,8 @@ func defaultConfig() (*Config, error) {
 			DefaultSubnetPools:        DefaultSubnetPools,
 			DefaultRootlessNetworkCmd: "slirp4netns",
 			DNSBindPort:               0,
-			CNIPluginDirs:             DefaultCNIPluginDirs,
-			NetavarkPluginDirs:        DefaultNetavarkPluginDirs,
+			CNIPluginDirs:             attributedstring.Slice{Values: DefaultCNIPluginDirs},
```

Comment:
maybe to reduce the boiler plate a bit add a function
`func newAttributedStringSlice(s []string) attributedstring.Slice`
and use that for all assignments.

Right now it may not look better but lets assume we end up adding a new field to struct and we want to set a non default value for all then it would be a one line change.